rke/k8s_rke_system_images.go

```diff
@@ -2654,37 +2654,37 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 		// Enabled out of band post v2.4.6
 		// Reminder: This template contains Nodelocal image which isn't in templates before v2.3.6
 		"v1.16.14-rancher1-1": {
-			Etcd:                      m("quay.io/coreos/etcd:v3.3.15-rancher1"),
+			Etcd:                      m("rancher/coreos-etcd:v3.3.15-rancher1"),
 			Kubernetes:                m("rancher/hyperkube:v1.16.14-rancher1"),
 			Alpine:                    m("rancher/rke-tools:v0.1.64"),
 			NginxProxy:                m("rancher/rke-tools:v0.1.64"),
 			CertDownloader:            m("rancher/rke-tools:v0.1.64"),
 			KubernetesServicesSidecar: m("rancher/rke-tools:v0.1.64"),
-			KubeDNS:                   m("gcr.io/google_containers/k8s-dns-kube-dns:1.15.0"),
-			DNSmasq:                   m("gcr.io/google_containers/k8s-dns-dnsmasq-nanny:1.15.0"),
-			KubeDNSSidecar:            m("gcr.io/google_containers/k8s-dns-sidecar:1.15.0"),
-			KubeDNSAutoscaler:         m("gcr.io/google_containers/cluster-proportional-autoscaler:1.7.1"),
-			Flannel:                   m("quay.io/coreos/flannel:v0.12.0"),
+			KubeDNS:                   m("rancher/k8s-dns-kube-dns:1.15.0"),
+			DNSmasq:                   m("rancher/k8s-dns-dnsmasq-nanny:1.15.0"),
+			KubeDNSSidecar:            m("rancher/k8s-dns-sidecar:1.15.0"),
+			KubeDNSAutoscaler:         m("rancher/cluster-proportional-autoscaler:1.7.1"),
+			Flannel:                   m("rancher/coreos-flannel:v0.12.0"),
 			FlannelCNI:                m("rancher/flannel-cni:v0.3.0-rancher6"),
-			CalicoNode:                m("quay.io/calico/node:v3.13.4"),
-			CalicoCNI:                 m("quay.io/calico/cni:v3.13.4"),
-			CalicoControllers:         m("quay.io/calico/kube-controllers:v3.13.4"),
-			CalicoCtl:                 m("quay.io/calico/ctl:v3.13.4"),
-			CalicoFlexVol:             m("quay.io/calico/pod2daemon-flexvol:v3.13.4"),
-			CanalNode:                 m("quay.io/calico/node:v3.13.4"),
-			CanalCNI:                  m("quay.io/calico/cni:v3.13.4"),
-			CanalFlannel:              m("quay.io/coreos/flannel:v0.12.0"),
-			CanalFlexVol:              m("quay.io/calico/pod2daemon-flexvol:v3.13.4"),
+			CalicoNode:                m("rancher/calico-node:v3.13.4"),
```

Comment:
This image `quay.io/calico/node:v3.13.4` and all the calico and canal images exist and shouldn't be updated.